backend/internal/server/server.go

```diff
@@ -16,6 +16,8 @@ import (
 	_ "github.com/jak103/powerplay/internal/server/apis/notifications"
 	_ "github.com/jak103/powerplay/internal/server/apis/stats"
 	_ "github.com/jak103/powerplay/internal/server/apis/user"
+	_ "github.com/jak103/powerplay/internal/server/apis/groups"
```

Comment:
I assume you saw we changed the file structure for this, but in case not we are putting seasons, leagues, teams, games, etc. into a folder so we don't have too many separate folders and they seem to fit nicely in this way